it/src/test/java/org/corfudb/universe/GenericIntegrationTest.java

```diff
@@ -0,0 +1,91 @@
+package org.corfudb.universe;
+
+import com.spotify.docker.client.DefaultDockerClient;
+import com.spotify.docker.client.DockerClient;
+import org.corfudb.universe.logging.LoggingParams;
+import org.corfudb.universe.scenario.Scenario;
+import org.corfudb.universe.scenario.fixture.Fixtures.AbstractUniverseFixture;
+import org.corfudb.universe.scenario.fixture.Fixtures.UniverseFixture;
+import org.corfudb.universe.scenario.fixture.Fixtures.VmUniverseFixture;
+import org.corfudb.universe.universe.Universe;
+import org.corfudb.universe.universe.Universe.UniverseMode;
+import org.corfudb.universe.universe.UniverseParams;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.rules.TestName;
+
+public abstract class GenericIntegrationTest {
```

Comment:
Rename this to AbstractIntegrationTest

Reply:
I wouldn't like to do it, because we already have AbstractIT, it will be another abstract test class